enterprise/migrations/0002_enterprisecustomerbrandingconfiguration.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
```

Comment:
I didn't originally, no. I looked now and the only thing I could find about `on_delete` was https://docs.djangoproject.com/en/1.11/releases/1.9/#foreignkey-and-onetoonefield-on-delete-argument -- which is basically what I'm covering here.